packages/components/textarea/src/Textarea.tsx

```diff
@@ -0,0 +1,24 @@
+import { Input, InputProps } from '@spark-ui/input'
+import { cx } from 'class-variance-authority'
+import { ComponentPropsWithoutRef, forwardRef, PropsWithChildren } from 'react'
+
+export interface TextareaProps
+  extends ComponentPropsWithoutRef<'textarea'>,
+    Pick<InputProps, 'intent'> {}
+
+export const Textarea = forwardRef<HTMLTextAreaElement, PropsWithChildren<TextareaProps>>(
+  ({ className, intent, disabled, rows = 1, ...others }, ref) => {
+    return (
+      <Input
+        className={cx(className, 'py-md resize-y')}
```

Comment:
I'm wondering if we should not add the `self-start` class here (`align-self: flex-start`) because the component renders differently when used as a standalone VS inside of a FormField, where it takes the full width available:
![Capture d’écran 2023-06-14 à 18 11 10](https://github.com/adevinta/spark/assets/2033710/b9f965ec-fc30-4975-b022-f93442a541cc)

WDYT ?